docs/index.mdx

```diff
@@ -1,18 +1,14 @@
 ---
 slug: /
-title: Home
+title: ｈａｃｋｍｕｄ wiki
```

Comment:
By default, this title will appear in the window title and as an \<h1> in the page.
This causes a weird window title of "hackmud wiki | hackmud wiki," because Docusaurus composes the page title with the site title.
![image](https://github.com/comcode-org/hackmud_wiki/assets/1775803/e9020c2b-7dbf-4428-843c-31191e435456)

Consider putting something better for the window title here (e.g. "Home"), and then manually putting a top-level heading in the content of the page, which will disable the automatic heading creation. E.g.:
\# ｈａｃｋｍｕｄ wiki

There's also a [`hide_title` frontmatter key](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#hide_title) if we want to accomplish this more explicitly.

Reply:
related https://github.com/comcode-org/hackmud_general/issues/27

Reply:
edited.

Reply:
I don't see any new commits.

Reply:
R I P -- looks like they didn't push?